app/src/newdocs/components/CanvasModuleConfig/index.jsx

```diff
@@ -0,0 +1,135 @@
+/* eslint-disable flowtype/no-types-missing-file-annotation */
+import React from 'react'
+import isEmpty from 'lodash/isEmpty'
+import ReactMarkdown from 'react-markdown'
+
+import createMdSnippet from '$newdocs/utils/createMdSnippet'
+import styles from './canvasModuleConfig.pcss'
+
+const inputs = (moduleInputs, moduleInputsRaw) => {
+    const types = []
+    const defaultValues = []
+
+    if (moduleInputs && moduleInputsRaw) {
+        Object.entries(moduleInputs).map(([key]) =>
+            moduleInputsRaw.map((ins) => {
```

Comment:
Strongly recommend always using `forEach` unless you're intending to use the return value from the `map`. `forEach` immediately says "I'm iterating and performing some side effect" while `map` says "I'm building a new array from this existing array".